modules/couchbase/src/test/java/org/testcontainers/couchbase/CouchbaseContainerTest.java

```diff
@@ -33,50 +33,39 @@
 
 public class CouchbaseContainerTest {
 
-    private static final DockerImageName COUCHBASE_IMAGE_ENTERPRISE = DockerImageName.parse(
-        "couchbase/server:enterprise-7.0.3"
-    );
+    private static final String COUCHBASE_IMAGE_ENTERPRISE = "couchbase/server:enterprise-7.0.3";
 
-    private static final DockerImageName COUCHBASE_IMAGE_COMMUNITY = DockerImageName.parse(
-        "couchbase/server:community-7.0.2"
-    );
+    private static final String COUCHBASE_IMAGE_ENTERPRISE_RECENT = "couchbase/server:enterprise-7.6.2";
 
-    @Test
-    public void testBasicContainerUsageForEnterpriseContainer() {
-        // bucket_definition {
-        BucketDefinition bucketDefinition = new BucketDefinition("mybucket");
-        // }
-
-        try (
-            // container_definition {
-            CouchbaseContainer container = new CouchbaseContainer(COUCHBASE_IMAGE_ENTERPRISE)
-                .withBucket(bucketDefinition)
-            // }
-        ) {
```

Comment:
These code snippets are used in our docs 😃. So, shouldn't be removed.

Reply:
Restored. Thanks!